index_mobile.html

```diff
@@ -41,11 +41,11 @@
 
 <!--========== HEADER ==========-->
 <!-- Start Navbar -->
-<nav class="navbar fixed-top navbar-expand-lg">
+<nav id="nav-bar" class="navbar fixed-top navbar-expand-lg">
```

Comment:
Can you please add the two `id`s in index.html as well? We'll eventually use one header file and one footer file for all pages.

Reply:
yeah ok!